### PR TITLE
Fix DMs showing players that have cryoed

### PR DESCRIPTION
**Pull request by JohnWildkins**
title

### CODE REVIEW

**Comment by JohnWildkins** [comment]
!review

File: code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm [hunk 1]
```diff
@@ -29,6 +29,11 @@
 	if(!comp)
 		return
 
+/datum/computer_file/program/chatclient/Destroy()
+	if(src in ntnet_global.chat_clients)
```
Comment (mikomyazaki) [inline code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm:33]: Don't need this check. If it isn't in `ntnet_global.chat_clients` then `-=` won't remove it.

File: code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm [hunk 2]
```diff
@@ -265,6 +270,9 @@
 		if((confirm != "Yes") || (CanUseTopic(usr) != STATUS_INTERACTIVE))
 			return FALSE
 
+	if(src in ntnet_global.chat_clients)
```
Comment (mikomyazaki) [inline code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm:273]: Same.

File: code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm [hunk 3]
```diff
@@ -293,7 +301,8 @@
 /datum/computer_file/program/chatclient/event_unregistered()
 	..()
 	computer.set_autorun(filename)
-	ntnet_global.chat_clients -= src
+	if(src in ntnet_global.chat_clients)
```
Comment (mikomyazaki) [inline code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm:304]: Same.

**Review by mikomyazaki — CHANGES_REQUESTED** [review]

File: code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm (hunk 1 above)
Comment (JohnWildkins) [reply code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm:33]: monke moment

File: code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm [hunk 4]
```diff
@@ -29,6 +29,10 @@
 	if(!comp)
 		return
 
+/datum/computer_file/program/chatclient/Destroy()
+	ntnet_global.chat_clients -= src
+	. = ..()
```
Comment (Geevies) [inline code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm:34]:
```suggestion
	return ..()
```

**Review by Geevies — APPROVED** [review]

File: code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm (hunk 4 above)
Comment (JohnWildkins) [reply code/modules/modular_computers/file_system/programs/generic/ntnrc_client.dm:34]: dang thats crazy

**Review by mikomyazaki — APPROVED** [review]

**Pull request merged**